packages/datadog_flutter_plugin/lib/src/tracing/tracing_headers.dart

```diff
@@ -46,66 +47,140 @@ class W3CTracingHeaders {
   static const tracestate = 'tracestate';
 }
 
-enum TraceIdRepresentation {
+/// Controls how we print a TracingId
+enum TracingIdRepresentation {
```

Comment:
/informational

are these types publicly exposed? If yes, it is breaking change. I don't think we are going to a major version but just to let customers.

Reply:
No they're not. They're only "publicly" exposed on the `datadog_internal.dart` portion of the package which is not part of our public interface.